src/get-started/codelab.md

```diff
@@ -1,842 +1,52 @@
-  is a recommended best practice for `State` objects.<br><br>
-  The IDE also automatically updates the state class
-  to extend `State<RandomWords>`, indicating
-  that you're using a generic [`State`][]
-  class specialized for use with `RandomWords`.
-  Most of the app's logic resides here&mdash;it maintains
-  the state for the `RandomWords` widget. This class saves the list
-  of generated word pairs, which grows infinitely
-  as the user scrolls and, in part 2 of this lab,
-  favorites word pairs as the user adds or removes them from
-  the list by toggling the heart icon.<br><br>
-  Both classes now look as follows:
-
-<?code-excerpt "lib/main.dart (RandomWordsWidget)" replace="/final wordPair = WordPair.random\(\);\n *return Text\(wordPair.asPascalCase\);/return Container();/g" indent-by="2"?>
-  ```dart
-  class RandomWords extends StatefulWidget {
-    const RandomWords({super.key});
-
-    @override
-    State<RandomWords> createState() => _RandomWordsState();
-  }
-
-  class _RandomWordsState extends State<RandomWords> {
-    @override
-    Widget build(BuildContext context) {
-      return Container();
-    }
-  }
-  ```
-</li>
-
-<li markdown="1"> Update the `build()` method in `_RandomWordsState`:
-  <?code-excerpt "lib/main.dart (_RandomWordsState)" title replace="/(\n  )(.*)/$1[!$2!]/g"?>
-  ```dart
-  class _RandomWordsState extends State<RandomWords> {
-    [!@override!]
-    [!Widget build(BuildContext context) {!]
-    [!  final wordPair = WordPair.random();!]
-    [!  return Text(wordPair.asPascalCase);!]
-    [!}!]
-  }
-  ```
-</li>
-
-<li markdown="1"> Remove the word generation code from `MyApp`
-  by making the changes shown in the following diff:
-
-  <?code-excerpt path-base="codelabs/startup_namer"?>
-  <?code-excerpt "{step2_use_package,step3_stateful_widget}/lib/main.dart" to="}"?>
-  ```diff
-  --- step2_use_package/lib/main.dart
-  +++ step3_stateful_widget/lib/main.dart
-  @@ -14,16 +14,15 @@
-
-     @override
-     Widget build(BuildContext context) {
-  -    final wordPair = WordPair.random();
-       return MaterialApp(
-         title: 'Welcome to Flutter',
-         home: Scaffold(
-           appBar: AppBar(
-             title: const Text('Welcome to Flutter'),
-           ),
-  -        body: Center(
-  -          child: Text(wordPair.asPascalCase),
-  +        body: const Center(
-  +          child: RandomWords(),
-           ),
-         ),
-       );
-     }
-  ```
-
-<li markdown="1"> Restart the app.
-  The app should behave as before, displaying a word
-  pairing each time you hot reload or save the app.
-</li>
+</div>
 
 {{site.alert.tip}}
-  If you see a warning on a hot reload that you might
-  need to restart the app, consider restarting it.
-  The warning might be a false positive, but
-  restarting your app ensures that
-  your changes are reflected in the app's UI.
+  The codelab above walks you through writing your first Flutter
+  app for all platforms &mdash; mobile, desktop and web.
```

Comment:
To be consistent with the other oxford comma you added :)

```suggestion
  app for all platforms &mdash; mobile, desktop, and web.
```

Reply:
Gah! I didn't make it in time, sorry!